airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +64,349 @@ def get_value(args):
         print(value)
     except AirflowConfigException:
         pass
+
+
+class ConfigChange:
+    """Class representing the configuration changes in Airflow 3.0."""
+
+    def __init__(
+        self, section: str, option: str, suggestion: str = "", renamed_to: tuple[str, str] | None = None
+    ) -> None:
+        """
+        Initialize a ConfigChange instance.
+
+        :param section: The section of the configuration.
+        :param option: The option within the section that is removed or deprecated.
+        :param suggestion: A suggestion for replacing or handling the removed configuration.
+        :param renamed_to: The new section and option if the configuration is renamed.
+        """
+        self.section = section
+        self.option = option
+        self.suggestion = suggestion
+        self.renamed_to = renamed_to
```

Comment:
We probably could make `renamed_to` a NamedTuple. At first glance, I thought it was (old_name, new_name), but it turned out to be (section, name)

Reply:
Changed it